src/World.h

```diff
@@ -1077,6 +1077,9 @@ class cWorld :
 	/** <summary>Generates a random spawnpoint on solid land by walking chunks and finding their biomes</summary> */
 	void GenerateRandomSpawn(void);
 
+	/** Check if player starting point is acceptable **/
+	bool cWorld::CheckPlayerSpawnPoint(int posX, int posY, int posZ);
```

Comment:
Also we start paramters with an `a_` prefix, and use PascalCase for the name. So `posX` would become `a_PosX` etc.